[Bug]: Test error

### Brief problem description

Run _uv run poe_ test  command and and see 
```diff
! FAILED tests/test_items.py::test_get_item_1 - assert 404 == 200

### Steps to Reproduce

Fix the code of function get_task_by_id

### Expected Result

Error will disappear

### Actual Result

Fixed

--- a/.vscode/settings.json
+++ b/.vscode/settings.json
@@ -26,5 +26,6 @@
   // Set these settings to "true" if you want
   // Markdown and preview to scroll synchronously.
   "markdown.preview.scrollEditorWithPreview": false,
-  "markdown.preview.scrollPreviewWithEditor": false
+  "markdown.preview.scrollPreviewWithEditor": false,
+  "python-envs.pythonProjects": []
 }

--- a/src/app/services/item_service.py
+++ b/src/app/services/item_service.py
@@ -228,7 +228,7 @@ def get_item_by_id_dfs_iterative(
 
                     for task in lab.tasks:
                         counter += 1
-                        if lab.id == item_id:
+                        if task.id == item_id:
                             return FoundItem(task, counter)
 
                         for step in task.steps:

